### DISCUSSION TITLE
Best mining pool for low-end PC

### DISCUSSION THREAD

**Discussion by DirtyVegas** [starcoinorg/starcoin#3133] [General]
Hello stars. What is the best mining pool to join for a low-end PC? I am using XMrig on poolin.me right now but it is slow going.

**Comment by DirtyVegas** [comment]
Nevermind. Found it. I'll try elephant. Only 1 I could find.

How can I run a node? Is it like DOGE where you can just install and run locally?

**Reply by livid** [reply]
Docker is probably the easiest way to run a local node if you have used Docker before.

It would need about 30GB of storage and one week of time for initial data sync.

**Reply by DirtyVegas** [reply]
Gotcha. I have not used docker before but I have the storage. Might as well give it a whirl. What is the plus side of running a local node? Any incentives?

**Reply by DirtyVegas** [reply]
And thank you for the response btw.

**Reply by jolestar** [reply]
> Gotcha. I have not used docker before but I have the storage. Might as well give it a whirl. What is the plus side of running a local node? Any incentives?

No incentives for running a local node currently.
